src/sharedServices/athena/createQuerySql.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { createQuerySql } from './createQuerySql'
+import {
+  testDataRequest,
+  noEventIdTestDataRequest,
+  dataPathsTestDataRequest
+} from '../../utils/tests/testDataRequest'
+
+describe('create Query SQL', () => {
+  test('returns a formatted SQL query string if all conditions satisfied', () => {
+    expect(createQuerySql(dataPathsTestDataRequest)).toEqual({
```

Comment:
Because you have different types of identifier, I'd like to see a parameterised test to try these different scenarios out. There's a Jest thing called `it.each`

Reply:
Done

Reply:
Better, but can you also parameterise the happy path? i.e. `returns a formatted SQL query string if all conditions satisfied` needs one for each type of id, because you specifically deal with each in the code